biomage/rotate_ci/rotate_ci.py

```diff
@@ -1,13 +1,15 @@
-import click
```

Comment:
the imports are automatically ordered by VS Code using the settings in `biomage-utils.code-workspace`. Either use replicate those settings in your IDE or move to VS Code

Reply:
I can see the line in the workspace but are you sure this is doing the right thing? The ordering seems to be random (not even alphabetic). I manually reordered them into the normal scheme (alphabetic as commanded in PEP8, followed by splitting into stdlib/third party/own groups as recommended by most style guides ([like this](https://github.com/reddit-archive/reddit/wiki/PythonImportGuidelines#ordering-of-import-statements)).

Reply:
mmm fair enough we'll need to have a look. but let's find some tool, doing it manually it's a mess as they will keep changing depending on commiter

Reply:
Yup, that sounds good to me.

Reply:
I added `isort` to the development requirements, and `make fmt` will run it, automatically sorting all imports.

Reply:
To be clear, from the changes to the workspace file I assume this is also automatically done when you save the files in VS Code manually, correct?

Reply:
Pol added this, but yes.